packages/discord.js/src/structures/Guild.js

```diff
@@ -737,8 +737,8 @@ class Guild extends AnonymousGuild {
    * @property {string} [name] The name of the guild
    * @property {VerificationLevel|number} [verificationLevel] The verification level of the guild
    * @property {ExplicitContentFilterLevel|number} [explicitContentFilter] The level of the explicit content filter
```

Comment:
This is also nullable. It resets the explicit media content filter back to not scanning anything (0).